source/appModules/devenv.py

```diff
@@ -88,6 +90,9 @@
 
 class AppModule(appModuleHandler.AppModule):
 	def chooseNVDAObjectOverlayClasses(self, obj, clsList):
+		vs_major, vs_minor, rest = self.productVersion.split(".", 2)

```

Comment:
```suggestion
		vsMajor, vsMinor = map(int, self.productVersion.split(".", 2)[:2])
```

You can also remove the lina below. Note the change from vs_major to vsMajor, vs_minor to vsMinor, which is more like the general code style of NVDA.